src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -52,22 +52,24 @@ export const ArtworkApp: React.SFC<Props> = props => {
           <ArtworkDetails artwork={props.artwork} />
         </Col>
       </Row>
-      <Row>
-        <Col sm={8}>
-          <ArtistInfo artistID={props.artwork.artist.id} />
-        </Col>
-      </Row>
-
-      {typeof window !== "undefined" && (
+      {props.artwork.artist && (
```

Comment:
Also looked at some Cultural maker artworks - those don't have artists and will crash here.

Technically I think there should be a test for this stuff but not sure if we have any of those yet for the top level page.

Reply:
We're just in the cleanup phase now, all of this stuff is gonna be moved around. Not ready to be tested (but will be tested soon).